bokeh/client/session.py

```diff
@@ -256,6 +261,7 @@ def close(self, why="closed"):
         self._connection.close(why)
 
     def loop_until_closed(self):
+        [cb.start() for cb in self._callbacks]
```

Comment:
start() doesn't do anything until the loop is running right? You could run the loop yourself or run it via loop_until_closed, but anytime it's running the callbacks should run.